src/containers/LearningResourcePage/components/LearningResourceForm.jsx

```diff
@@ -65,6 +65,7 @@ export const getInitialValues = (article = {}) => {
     updated: article.updated,
     grepCodes: article.grepCodes || [],
     conceptIds: article.conceptIds || [],
+    availability: article.availability || 'everyone',
```

Comment:
Bruke enum?

Reply:
Syntes enum blir unødvendig når union fungerer fint.

Reply:
Får ikke isolere èn enkelt verdi uten å definere typen. I .jsx filer har man ikke typer så denne måten å skrive på er less prone to error:
```
availability: article.availability || AvailabilityType.EVERYONE,
```

Reply:
Ikke enig.
AvailabilityType.EVERYONE er nøyaktig det samme som 'everyone'
Og typescript klager om du skriver noe som ikke er i unionen

Reply:
Men nå snakker vi i JSX filer (`LearningResourceForm.jsx`, `TopicArticleForm.jsx`) der er det ingen typescript som kan klage på oss dersom vi innfører skrivefeil

Reply:
Sant nok, men alle filer med PropTypes har jo problemet med manglende detaljert typebeskrivelse